test_unstructured/partition/test_msg.py

```diff
@@ -121,17 +105,9 @@ def test_partition_msg_from_file_with_metadata_filename():
         assert element.metadata.filename == "test"
 
 
-def test_extract_attachment_info():
-    filename = example_doc_path("fake-email-attachment.msg")
-    attachment_info = extract_msg_attachment_info(filename)
-    assert len(attachment_info) > 0
-    assert attachment_info == ATTACH_EXPECTED_OUTPUT
```

Comment:
`extract_attachment_info()` is no longer used. Those fields are directly available from `oxmsg.Attachment`.